Tasks/AzureCLIV2/azureclitask.ts

```diff
@@ -46,7 +46,10 @@ export class azureclitask {
             }
         }
         finally {
-            await scriptType.cleanUp();
+            if (scriptType) {
```

Comment:
in yaml case, if scriptType is not set, should we throw an error and fail the task?

Reply:
Yes, that is the behaviour even today. But it was being overriden by the reference error being generated from the finally block. 

See this line in ScriptType.ts: 
_let scriptType: string = tl.getInput("scriptType", true);_